hack/code/instancetype_testdata_gen/main.go

```diff
@@ -63,8 +63,11 @@ func main() {
 		log.Fatalf("setting AWS_REGION, %s", err)
 	}
 	ctx := context.Background()
-	sess := session.Must(session.NewSession())
-	ec2Client := ec2.New(sess)
+	cfg, err := config.LoadDefaultConfig(ctx)
```

Comment:
We should opt for `lo.Must` in a lot of these places where we are going to throw panics anyways